src/views/SignIn.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="row">
+  <div class="signIn w-100">
+    <form>
+      <b-col>
+        <b-form-group id="input-group-first-name" label="Your Firstname:" label-for="input-first-name">
+          <b-form-input
+            id="input-first-name"
+            class="w-100 input-styling"
+            type="text"
+            v-model="signIn.firstName"
+            required
+            placeholder="Max">
+          </b-form-input>
+        </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-last-name" label="Your Lastname:" label-for="input-last-name">
+        <b-form-input
+          id="input-last-name"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.lastName"
+          required
+          placeholder="Mustermann">
+        </b-form-input>
+      </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-initials" label="Your Initials:" label-for="input-initials">
+        <b-form-input
+          id="input-initials"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.initials"
+          required
+          placeholder="mmn"></b-form-input>
+      </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-master-password" label="Master Password:" label-for="input-master-password">
+        <b-form-input
+          id="input-master-password"
+          type="password"
+          autocomplete="on"
+          class="w-100 input-styling"
+          v-model="signIn.masterPassword"
+          required
+          placeholder="Enter master password"
+          ></b-form-input>
+      </b-form-group>
+      </b-col>
+      <b-col>
+        <b-button class="w-100 mt-4 sy-background signIn-button" @click="onSubmit">Sign in</b-button>
```

Comment:
Sicher das überall dieses w-100 gebraucht wird?
Wenn man erreichen möchte das ein Button den ganzen Platz ein nimmt, könnte man auch d-block geben
```suggestion
        <b-button class="d-block mt-4 sy-background signIn-button" @click="onSubmit">Sign in</b-button>
```